src/map/status.cpp

```diff
@@ -7549,6 +7549,8 @@ static short status_calc_aspd(struct block_list *bl, struct status_change *sc, b
 			bonus += sc->data[SC_INCREASEAGI]->val1;
 		if (sc->data[SC_NIBELUNGEN] && sc->data[SC_NIBELUNGEN]->val2 == RINGNBL_ASPDRATE)
 			bonus += 20;
+		if (sc->data[SC_STARSTANCE])
```

Comment:
Just confirming if its appropriate to add this bonus same with pre-renewal (status_calc_aspd) even it doesn't have its class implemented there. For the sake of others who are recompiling them at pre-re.

Reply:
I believe there already is a condition in place for Pre-renewal but I haven't tested it personally.
See `status_calc_aspd_rate`:
https://github.com/rathena/rathena/blob/132180325435847a9470f82cb795525d03be1b67/src/map/status.cpp#L7742-L7743

Reply:
Ah yeah. overlooked at it I thought both pre-re and re are not having its bonuses. I stand corrected.